docs/technical-reference/api/kyma-cr.md

```diff
@@ -103,11 +103,15 @@ In addition to this very flexible way of referencing modules, there is also anot
 While `CreateAndDelete` will cause the ModuleTemplate's **.spec.data** to be created and deleted to initialize a module with preconfigured defaults, `Ignore` can be used to only initialize the operator without initializing any default data.
 This allows users to be fully flexible in regard to when and how to initialize their module.
 
+### **.spec.modules[].remoteModuleTemplateRef**
+The `remoteModuleTemplateRef` flag allows the users to have their ModuleTemplate fetched from the SKR cluster instead of the control-plane. It should be the reference (FQDN,
```

Comment:
```suggestion
The `remoteModuleTemplateRef` flag allows the users to have their ModuleTemplate CR fetched from the SKR cluster instead of Kyma Control Plane (KCP). It should be the reference (FQDN,
```